### ISSUE TITLE
Fidelity stock price issue. 

### ISSUE THREAD

**Issue by ImNotOssy** [NelsonDane/auto-rsa#279]
looks like Fidelity is one of those few brokerages that have issues, although this issue is minor, it's a little annoying.
when running !rsa holdings Fidelity, the stocks show up but its without the amount held and prices.

![image](https://github.com/NelsonDane/auto-rsa/assets/85385882/d4946d01-fed4-4300-8882-6c8678acc88a)

**Comment by NelsonDane** [comment]
Unfortunately, this is the intended behavior for now. The way that Fidelity formats that data on their site makes it incredibly difficult to scrape and I didn't have the time to dedicate to that :(

**Comment by ImNotOssy** [comment]
> Unfortunately, this is the intended behavior for now. The way that Fidelity formats that data on their site makes it incredibly difficult to scrape and I didn't have the time to dedicate to that :(

Ah, i figured, at least i can see what stocks i have and still sell properly ,i just have to make sure its the correct price so i can sell them.

**Comment by NelsonDane** [comment]
Yeah, it's on the back burner, but unfortunately other stuff always comes up haha

**Comment by ImNotOssy** [comment]
> Yeah, it's on the back burner, but unfortunately other stuff always comes up haha

is there anything on why Fidelity doesn't work on the docker? works fine when doing through CLI.
![image](https://github.com/NelsonDane/auto-rsa/assets/85385882/013eda16-9672-4c2e-a15b-43a6d12bd18e)

**Comment by PZES** [comment]
I believe that an easier way is to download the csv from https://digital.fidelity.com/ftgw/digital/portfolio/positions and parse it for all the info
